resources/shellHelperFunctions.sh

```diff
@@ -281,13 +281,17 @@ verify_npm_setup() {
 # see /docs/build/node.md
 _select_node_version_with_nvm() {
   local REQUIRED_NODE_VERSION="$("$JQ" -r '.engines.node' "$KEYMAN_ROOT/package.json")"
+  local CURRENT_NODE_VERSION
 
   if [[ $BUILDER_OS != win ]]; then
     # launch nvm in a sub process, see _builder_nvm.sh for details
     "$KEYMAN_ROOT/resources/build/_builder_nvm.sh" "$REQUIRED_NODE_VERSION"
   else
-    nvm install "$REQUIRED_NODE_VERSION"
-    nvm use "$REQUIRED_NODE_VERSION"
+    CURRENT_NODE_VERSION="$(node --version)"
+    if [[ "$CURRENT_NODE_VERSION" != "v$REQUIRED_NODE_VERSION" ]]; then
+      nvm install "$REQUIRED_NODE_VERSION"
+      start /wait nvm use "$REQUIRED_NODE_VERSION"
```

Comment:
```suggestion
      start //wait //b nvm install "$REQUIRED_NODE_VERSION"
      start //wait //b nvm use "$REQUIRED_NODE_VERSION"
```

For consistency, I think we should wait on both commands? Also add `/b` param to use same console window (and double-slash for bash escaping to prevent conversion to `\`).